recipes/inotify_simple/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "inotify_simple" %}
+{% set version = "1.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: fc2c10dd73278a1027d0663f2db51240af5946390f363a154361406ebdddd8dd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [ win ]
```

Please remove the spaces around the selector

```suggestion
  skip: true  # [win]
```